### PR TITLE
Nuke: add pointcache and animation to loader

### PR DESCRIPTION
**Pull request by gabormarinov**
## Brief description
Added pointcache and animation families to model loader

### CODE REVIEW

File: openpype/hosts/nuke/plugins/load/load_model.py [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
-    This will load alembic model into script.
+    This will load alembic model or anim into script.
     """
 
-    families = ["model"]
+    families = ["model", "pointcache", "animation"]
     representations = ["abc"]
 
-    label = "Load Alembic Model"
+    label = "Load Alembic Model or Anim"
```
Comment (BigRoy) [inline openpype/hosts/nuke/plugins/load/load_model.py:24]: Might this be nicer if it's just more generic?
```suggestion
    label = "Load Alembic"
```

Comment (gabormarinov) [reply openpype/hosts/nuke/plugins/load/load_model.py]: Agreed :)

**Comment by iLLiCiTiT** [comment]
Please make a different branch name for future PRs. It makes it harder to review when the branch we should look for, is called `develop` :)

**Review by m-u-r-p-h-y — APPROVED** [review]
loading alembic models and pointcache sequences works fine 

tested in Nuke 12.2v9

image

**Pull request merged**